.travis.yml

```diff
@@ -30,6 +30,7 @@ script:
     - cd docs && make html && cd ..
     - conda install doctr
     - doctr deploy --deploy-repo ergs/rickshaw-docs .
+    - pytest -v
```

Comment:
This line should likely be above the doc creation and below the `set -e` line. We want the tests to fail first.